src/tests/D3MHub.t.sol

```diff
@@ -1699,6 +1640,263 @@ contract D3MHubTest is DSSTest {
         assertEq(art, 60 * WAD);
     }
 
+    function test_exec_fixInk_full_amount() public {
+        _windSystem();
+        // interest is determined by the difference in gem balance to dai debt
+        // by giving extra gems to the Join we simulate interest
+        _giveTokens(TokenLike(address(testGem)), 10 * WAD);
+        testGem.transfer(address(d3mTestPool), 10 * WAD); // Simulates 10 WAD of interest accumulated
+        assertEq(testGem.balanceOf(address(d3mTestPool)), 60 * WAD);
+        assertEq(dai.balanceOf(address(testGem)), 50 * WAD);
+
+        vat.file(ilk, "line", 55 * RAD);
+
+        (uint256 ink, uint256 art) = vat.urns(ilk, address(d3mTestPool));
+        assertEq(ink, 50 * WAD);
+        assertEq(art, 50 * WAD);
+        uint256 prevDai = vat.dai(vow);
+
+        d3mHub.exec(ilk);
+
+        (ink, art) = vat.urns(ilk, address(d3mTestPool));
+        assertEq(ink, 50 * WAD);
+        assertEq(art, 50 * WAD);
+        assertEq(vat.dai(vow), prevDai + 10 * RAD);
+        assertEq(testGem.balanceOf(address(d3mTestPool)), 50 * WAD);
+    }
```

Comment:
Do you think it's worth it to add a second exec call to these tests and make sure nothing has changed?

Reply:
I'm not really opposed to do so, but shouldn't we really do that everywhere.
This might be more interesting to do with Certora probably.

Reply:
Yeah, it's a bit ugly to add it everywhere.
If we cover it with Certora (or even just try it locally) it's also fine from my side.

Reply:
I tried it locally for all these types of tests, no failures.